### DISCUSSION TITLE
Note / Info / Tips etc

### DISCUSSION THREAD

**Discussion by markusmoenig** [McShelby/hugo-theme-relearn#495] [Q&A] [answered]
Hi,

in the website of this project there are special widets in the content showing Notes / Infos etc. I could not find references to them in the docs. Are these available in the basic theme or somehow done differently ?

Thanks

**Comment by McShelby** [comment, marked as answer]
Do you mean this? https://mcshelby.github.io/hugo-theme-relearn/shortcodes/notice/index.html

**Comment by markusmoenig** [comment]
Yes!

Sorry  overlooked it somehow!
